config/prod/agency_endpoints.json

```diff
@@ -271,9 +271,9 @@
   "website": "https://archives.gov/",
   "codeUrl": "https://archives.gov/code.json",
   "requirements": {
-    "agencyWidePolicy": 1,
-    "openSourceRequirement":0.5,
-    "inventoryRequirement":0.5
+    "agencyWidePolicy": null,
```

Comment:
Why are these set to null while the others are set to 0? Is it because archives.gov is not part of the CFO agencies?

Reply:
Yes. I discovered setting them to `null` makes them not included in the FE. Have not exactly found out what performs that magic.